src/pages/basic_data/app_admin/CustomerInstitutionMetadataForm.tsx

```diff
@@ -155,30 +156,29 @@ export const CustomerInstitutionMetadataForm = ({
 
             <Field name={CustomerInstitutionFieldNames.Sector}>
               {({ field }: FieldProps) => (
-                <div>
-                  <FormLabel component="legend" sx={{ fontWeight: 'bold' }}>
-                    {t('basic_data.institutions.sector')}
-                  </FormLabel>
-                  <Box sx={{ display: 'flex', gap: '0.5rem', mt: '0.5rem' }}>
-                    {Object.values(Sector).map((sector) => (
-                      <Chip
-                        key={sector}
-                        data-testid={dataTestId.basicData.institutionAdmin.sectorChip(sector)}
-                        label={t(`basic_data.institutions.sector_values.${sector}`)}
-                        color="primary"
-                        variant={field.value === sector ? 'filled' : 'outlined'}
-                        onClick={() => setFieldValue(field.name, sector)}
-                      />
-                    ))}
-                  </Box>
-                </div>
+                <TextField
+                  {...field}
+                  select
+                  fullWidth
+                  required
+                  label={t('basic_data.institutions.sector')}
+                  variant="filled">
+                  {Object.values(Sector).map((sector) => (
+                    <MenuItem
+                      key={sector}
+                      value={sector}
+                      data-testid={dataTestId.basicData.institutionAdmin.sectorChip(sector)}>
+                      {t(`basic_data.institutions.sector_values.${sector}`)}
+                    </MenuItem>
+                  ))}
+                </TextField>
               )}
             </Field>
 
-            <div>
+            <Box sx={{ display: 'flex', justifyContent: 'flex-start', gap: '1.5rem' }}>
               <Field name={CustomerInstitutionFieldNames.NviInstitution}>
                 {({ field }: FieldProps<boolean>) => (
-                  <>
+                  <Box>
```

Comment:
Om dette bare er for å gruppere det un der ett element, kan vi bruke en native `<div>` i stedet for MUI stuff
```suggestion
                  <div>
```